app/src/pages/ProfileEdit.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from "react"
```

Comment:
hi i know this seems minor but is profile edit a confusing name for this? this isn't necessarily part of an edit flow once a profile has been built. just want to avoid confusion! @martyboggs @tylerthome